Update it-IT.com_fabrik_admin.ini in transiflex

tanolalano opened the pull request:
As it was urgent as I finish the translation of string.
More IT translate coming soon

jaanusnurmoja committed:
ATM the only way to make correct urls in related data links...
in case the related element is not in main table but in a joined group

jaanusnurmoja committed:
Correct links to related data (form/details) +whoops

pollen8 commented:
hi - are you sure you mean to be doing this PR? The pollen8/fabrik repository isn't really used for anything

tanolalano commented:
I mean on the main repository...fabrik/fabrik
Apologyze

> Il giorno 22/ott/2013, alle ore 01:20, Rob Clayburn notifications@github.com ha scritto:
> 
> hi - are you sure you mean to be doing this PR? The pollen8/fabrik repository isn't really used for anything
> 
> —
> Reply to this email directly or view it on GitHub.

jaanusnurmoja committed:
related data links: code style & cleanup

cheesegrits committed:
Adding stuff to gitignore

Commit:
Merge branch 'master' of https://github.com/Fabrik/fabrik

jaanusnurmoja committed:
related links - whoops

Absence of $f (null instead) seems not to affect to related add, but it was originally there. Forgotten to restore it after one test with no result.

pollen8 committed:
xml file updates;

pollen8 committed:
added: form module option to show success message as alert, or inline at top of form

pollen8 committed:
fixed: form model getElementData didnt work for repeat elements such as ajax fileuploads

pollen8 committed:
Merge branch 'master' of github.com:Fabrik/fabrik

cheesegrits committed:
Added hiding message container when showing error container in form JS

cheesegrits committed:
Added 'bypass all email' option to juser, so we don't send any emails,
regardless of activation settings, etc.

Commit:
Merge branch 'master' of https://github.com/Fabrik/fabrik

Commit:
[FIX] v2 had the feature to completely hide an empty table. Restored this and implemented over class

pollen8 committed:
Merge pull request #1040 from paulv888/empty_table

Empty table

pollen8 committed:
fixed: issue when saving J article in admin when placing {fabrik view=list} with no layout property. Was giving a 500 error with layout default_edit not found error message

pollen8 committed:
Merge branch 'master' of github.com:Fabrik/fabrik

Commit:
Update template_css.php

The default for this should be to display. I will hide tables, where "filter required" = true

pollen8 committed:
added: title to related data view link

pollen8 committed:
Merge pull request #1044 from paulv888/empty_table

Update template_css.php

pollen8 committed:
Merge pull request #1037 from jaanusnurmoja/related_data_links_form_view

related links J2.5/3.0 - whoops

pollen8 committed:
changed: autofill plugin - can use placeholders in "from" may keys

pollen8 committed:
codestyle

pollen8 committed:
search all with empty date could produced date format error

pollen8 committed:
db join include in search all method needed updating

pollen8 committed:
update article form plugin

pollen8 committed:
update article list plugin

pollen8 committed:
fixed: isnull prefilter on dropdown with no label selected

pollen8 committed:
changes to date onchange evnet code - reduces unwanted additional onchange calls

pollen8 committed:
codestyle

pollen8 committed:
update rest plugin

pollen8 committed:
fixed: fileupload - edit record, add ajax files, delete, add a new file, all files (including deleted ones) were saved to the db

pollen8 committed:
fixed: fileupload - ajax upload, delete file uploaded during current edit, there is no record stored in the joined db yet, but we were deleteing based on the upload file id. As thats a stirng and we cased to int, we could indirectly delete other rows file uploads

pollen8 committed:
lang update

apzentral committed:
Merge remote-tracking branch 'upstream/master'

pollen8 committed:
fixed: email plugin, dbjoin in repeat group. Placeholders were returning raw data and not labels

pollen8 committed:
codestyle

pollen8 committed:
codestyle and set parseMessageforPlaceholder to accept arrays (as per 3.1)

pollen8 committed:
added: IS EMPTY option to advanced search

pollen8 committed:
backported element model getListHeading() method from 3.1

cheesegrits committed:
Addig limit param to fusion chart viz, used in $listModel->setLimits()
call.

cheesegrits committed:
Commenting out some test code in join element

pollen8 committed:
fixed: regression where db joins were rendering values in list and not labels

pollen8 committed:
fixed list nav page buttons not aligned

trob committed:
as in 3.1: set_ini of user agent, sometimes needed when calling file_get_content…  …s on https servers

trob committed:
as in 3.1:if using sync on edit, then re-editing the form would populate with original JFactory session user object, not loading the data previously update by the from

cheesegrits committed:
Merge branch 'master' of ssh://github.com/Fabrik/fabrik

cheesegrits committed:
Fix for getEmailDate, Rob had un-barfed one too many lines in a fix to a
fix.  You'd have to read the code to see what I mean.  :)

Commit:
Fixed: Trashed tables loaded in $aTableNames

pollen8 committed:
codestyle

pollen8 committed:
transifex config

pollen8 committed:
backported calcs setStoreDatabaseFormat from 3.1, date elements (with time selector) were inserting the wrong placeholder data

pollen8 committed:
Merge branch 'master' of github.com:Fabrik/fabrik

pollen8 committed:
removed disabled property from cdd with 0 options - just use class name

cheesegrits committed:
Fix of a fix to a fix to a fix of a fix in getEmailData

cheesegrits committed:
Merge branch 'master' of ssh://github.com/Fabrik/fabrik

pollen8 committed:
codestyle

pollen8 committed:
Merge branch 'master' of github.com:Fabrik/fabrik

pollen8 committed:
fixed: paypal plugin - subscription swtich not being applied

pollen8 committed:
enabled IN conditions in list query string filters

pollen8 committed:
possible fix for corner case paypal issue, where unconfirmed addresses would overwrite txn details with blank data

pollen8 committed:
Merge pull request #1079 from paulv888/trashed_tables

Fixed: Trashed tables loaded in $aTableNames

pollen8 committed:
backported email form plugin attach as pdf option (seems to attach the form rather than details view though) - added pdf helper - set do pdf tmp path to joomla temp path; could help when in safemode

pollen8 committed:
Merge branch 'master' of github.com:Fabrik/fabrik

pollen8 committed:
update dompdf

pollen8 committed:
update dompdf

pollen8 committed:
dompdf

pollen8 committed:
dompdf fonts

pollen8 committed:
dompdf

cheesegrits committed:
Fixing google chart viz to use getScheme() for Google URL.

cheesegrits committed:
Merge branch 'master' of ssh://github.com/Fabrik/fabrik

pollen8 committed:
pay fix for not overwriting ipn transaction details with empty data

pollen8 committed:
Merge branch 'master' of github.com:Fabrik/fabrik

pollen8 committed:
revert changes to chart vis under https as apparently google doesnt accept it

pollen8 committed:
updating rating element to be in line with 3.1, removed png/gif option

pollen8 committed:
fixed: decoding &#039; to apostrophe in filter list data

pollen8 committed:
fixed: error when a linked element was using access, "or use field" - in the child form we were looking up the parent forms element id - and of course it wasnt found

pollen8 committed:
fixed: juser email not sending as html;

pollen8 committed:
added {$mosConfig_secret} placeholder

pollen8 committed:
remove <pre>

pollen8 committed:
added: backend csv export view#

pollen8 committed:
tweak in img icon ordering loading (for podion)

pollen8 committed:
codestyle

pollen8 committed:
update col fixes

cheesegrits committed:
Adding locking of comments for JComments in comments form plugin.

cheesegrits committed:
Merge branch 'master' of ssh://github.com/Fabrik/fabrik

pollen8 committed:
fixing calcs on various custom plugins

pollen8 committed:
Merge branch 'master' of github.com:Fabrik/fabrik

pollen8 committed:
fixed: update col plugin - wasnt reporting the number of emails sent

cheesegrits committed:
Added placeholders to form success msg

pollen8 committed:
fixed: groupby could include blank groupby values in the same grouping as those with recorded data

pollen8 committed:
added: {subject} placeholder to list email update values

pollen8 committed:
cdd as autocomplete, if single value found automatically set that value in the cdd

cheesegrits committed:
Merge branch 'master' of ssh://github.com/Fabrik/fabrik

cheesegrits committed:
Attempting to fix issues with usergroup element and juser plugin, with
usergoup memberships getting overwritten when editing a form.

pollen8 committed:
resolved

pollen8 committed:
remove console from rating js

pollen8 committed:
lang update

pollen8 committed:
Merge branch 'master' of github.com:Fabrik/fabrik

pollen8 committed:
lang

pollen8 committed:
lang update

pollen8 committed:
Merge pull request #862 from Ajyamal/patch-2

Update ja-JP.com_fabrik.ini

pollen8 committed:
lang updates

pollen8 committed:
Merge branch 'master' of github.com:Fabrik/fabrik

pollen8 committed:
lang upates

pollen8 committed:
lang update, and new script to create transifex config file

pollen8 committed:
added transfex language config file generator

pollen8 committed:
Merge branch 'apzentral-master'

pollen8 committed:
fixed: csv export (with include filters option selected) was not taking into account advanced search values

pollen8 committed:
backporting fix for calc in inlineedit

pollen8 committed:
backported group by fix from 3.1

trob committed:
Typ: "_QQ_" crept in again

pollen8 committed:
rounded form tmpl updated;

pollen8 committed:
Merge branch 'master' of github.com:Fabrik/fabrik

cheesegrits committed:
Fix to remove elements from Advanced Search field list if
!canView('list').

cheesegrits committed:
Merge branch 'master' of ssh://github.com/Fabrik/fabrik

cheesegrits committed:
Fix to listcsv plugin, so returning false skips row

Sophist-UK committed:
Fix #1110 F3.0

Fix #1110 F3.0

njbair committed:
Added .js files to install manifest

After a clean install using 3.1rc2, I received a JavaScript console error (*404 plugins/fabrik_element/display/display-min.js* Not Found). The files are there, they just weren't installed because they were not listed in the manifest XML file. This should fix it.

pollen8 committed:
codestyle

pollen8 committed:
Merge branch 'master' of github.com:Fabrik/fabrik

pollen8 committed:
lang

pollen8 committed:
lang

pollen8 committed:
fixed: db error in processToDb - incorrect var $listDb used

pollen8 committed:
fixed: print button not showing in print preview page

pollen8 committed:
fixed: search all, then search all again would search on both terms, not replacing the first term with the second

pollen8 committed:
fixed: search all, then search all again would search on both terms, not replacing the first term with the second

pollen8 committed:
Merge branch 'master' of github.com:Fabrik/fabrik

pollen8 committed:
woops

pollen8 committed:
lang

pollen8 committed:
Merge pull request #1115 from njbair/patch-1

Added .js files to install manifest

pollen8 committed:
Merge pull request #1111 from fratac/master

Update timeline.xml

pollen8 committed:
Merge pull request #1113 from Sophist-UK/fix_fabrik_forms_log_issue_1110_f30

Fix #1110 F3.0

pollen8 committed:
lang

pollen8 committed:
Merge branch 'master' of github.com:Fabrik/fabrik

pollen8 committed:
lang

cheesegrits committed:
Adding warning to join element join clause language.  Does not work in
list view.  Feature deprecated.

pollen8 committed:
back ported data-fabrik-view code to load either form or list in popup window - applied to related links to fix fact they were not resizing on load

pollen8 committed:
bringing some updates from 3.1 back into 3.0

cheesegrits committed:
Fixing unNumberFormat() issue after failed validation for field element.

cheesegrits committed:
Merge branch 'master' of ssh://github.com/Fabrik/fabrik

pollen8 committed:
Merge branch 'master' of github.com:Fabrik/fabrik

pollen8 committed:
lang

pollen8 committed:
backport radiobutton - create options from php code

pollen8 committed:
fixed: backported elementlist filter logic - should now mean checkbox/mutliselect list filters work correctly on elements such as dropdowns

pollen8 committed:
fixed: dbjoin element in repeat group, getEmaildata incorrect -meant no data entered into the email msg

pollen8 committed:
added search plugin option to show/hide the list tile in search results

pollen8 committed:
removed incorrect input filter on list intro text

Sophist-UK committed:
F30 - Fix strings ending in ";

pollen8 committed:
Merge pull request #1167 from Sophist-UK/fix_strings_ending_in_semicolon_f30

F30 - Fix strings ending in ";

Commit:
Update fr-FR.com_fabrik.ini

pollen8 committed:
back porting some articles plugins stuff

pollen8 committed:
adding data-filter-row properties to list filter rows

pollen8 committed:
Merge branch 'master' of github.com:Fabrik/fabrik

pollen8 committed:
article form plugin - format any ajax upload file for inclusion in article content, regarless of whether its selected and a main/intro image

pollen8 committed:
fixed: date element - timeselect built with html: classname="foo" and not class="foo"

pollen8 committed:
added: reset filters option to list model

pollen8 committed:
added: reset filters option to list model

pollen8 committed:
backported readmore into article

pollen8 committed:
fixed: bday element in email - wasnt using details formatting option

cheesegrits committed:
Fixing QQ issue in update_col en-GB language

pollen8 committed:
fixed: J search, php error as params handled differently in j2.5 to j3.1

pollen8 committed:
fixed: view/edit icon overrides not searching current template folder

pollen8 committed:
article lang desc/label wrong way round

pollen8 committed:
codestyle/spelling

pollen8 committed:
pdf fonts

pollen8 committed:
article plugin update

pollen8 committed:
fixed: corner case - list based on view, with calcs (sum/avg etc) if pk set to field which contains non-unique data then certain rows could be exculded from the sum, producing a total too small

pollen8 committed:
fixed: sum query incorrect if using view as list base, and no distinct primary key selected

pollen8 committed:
fixed: error if attaching pdf in email plugin, and emailing to two addresses

pollen8 committed:
added: csv export encoding option

pollen8 committed:
Merge pull request #1171 from tportier/patch-1

Update fr-FR.com_fabrik.ini

pollen8 committed:
fixed: if dbjoin element isJoin() then filter options not ordered

pollen8 committed:
Merge branch 'master' of github.com:Fabrik/fabrik

pollen8 committed:
codestyle and a couple of article plugin backports

pollen8 committed:
codestyle spelling

pollen8 committed:
codestyle spelling

pollen8 committed:
hacky fix for advanced search on date >/< then csv export

pollen8 committed:
possible fix for csv export with advanced filter > date, also spelling and codestyle

pollen8 committed:
fixed: apos in custom link not encoded

pollen8 committed:
spelling

pollen8 committed:
encode links

pollen8 committed:
fixed: ajax validation no longer useses full url - could result in cross origin server errors

pollen8 committed:
admin nav not working, code style, plus backporting several things from 3.1 list model etc

pollen8 committed:
tweaking & reverting some of the last changes

pollen8 committed:
notices

pollen8 committed:
woops

pollen8 committed:
backport autocomplete from 3.1 - fixes issue where none-matched search text was being submitted - meant notempty validation wouldnt fail

pollen8 committed:
fixed: fileupload broke, plus code style changes

pollen8 committed:
removed warning

pollen8 committed:
fixed: form submission in ie could cause an error if form action contained a single quote character

pollen8 committed:
backported element emptyFilterValue from 3.1

pollen8 committed:
codestyle

pollen8 committed:
fixed: advanced search not converting string dates into sql format before submitting

pollen8 committed:
fixed: image element - single file name stored (so no path) and front end select set - should now wokr

pollen8 committed:
fixed: advanced search, filter once on > date element, submit, add 2nd filter on < same date element - gave error as 2nd js filter object not created/setup correctly

cheesegrits committed:
Back porting form article plugin fix for url safe aliases

cheesegrits committed:
Added safeQuote helper, which db vaue quotes an array of strings (or a
single string), and safeQuoted some keys in join element code.

cheesegrits committed:
Fixing merge conflict in article form plugin

Merge branch 'master' of ssh://github.com/Fabrik/fabrik

Conflicts:
	plugins/fabrik_form/article/article.php

Sophist-UK committed:
Fix #1158 for F30

Sophist-UK committed:
Address felixcat comments.

pollen8 committed:
fixed: fileupload - cropicon not showing

pollen8 committed:
Merge branch 'master' of github.com:Fabrik/fabrik

cheesegrits committed:
Back porting fix for character count periodical in textarea from 3.1

cheesegrits committed:
Merge branch 'master' of ssh://github.com/Fabrik/fabrik

pollen8 committed:
Merge pull request #1199 from Sophist-UK/enhance_link_element_f30

Fix #1158 for F30

cheesegrits committed:
Added missing icon in faceted link

cheesegrits committed:
Merge branch 'master' of ssh://github.com/Fabrik/fabrik

cheesegrits committed:
In map viz, strip media path on data icon so it doesn't get prepended twice

cheesegrits committed:
Added default location to map viz, for when no data, or centering method
doesn't return a valid point.

cheesegrits committed:
Fixing little issue with last commit on map center default

cheesegrits committed:
Fixed my schoolboy mistake in safeQuote() string helper, so it'll work
with embedded quotes, like O'Reilly.

cheesegrits committed:
Added $commaSeparated arg to safeQuote() string helper, so if passed a
string of comma separated values, will individually quote each value.

trob committed:
Typo $emailColID; $emailField must be

table___element, not table.element

cheesegrits committed:
Fixing issue in auto complete join, with multiple joins to same table on
one form.

cheesegrits committed:
Merge branch 'master' of ssh://github.com/Fabrik/fabrik

cheesegrits committed:
Adding GROU_CONCAT_MAX_LEN setting to 10240 in getDbo()

cheesegrits committed:
Setting GROUP_CONCAT_MAX_LEN to 10240 in system plugin

cheesegrits committed:
Added update() method to image element JS

cheesegrits committed:
Trimming root folder from image link in list display

cheesegrits committed:
Fixing issue with S3 adapter cleanName() that caused S3 uploads to
silently fail.  It was assuming randomizeFilename() returned the arg,
but it doesn't, it modifies a call by reference.

cheesegrits committed:
Added tableStyle to base list view, fixes a notice in 3.0, makes it
easier to share admin list tmpl code with 3.1 bootstrap tmpl

cheesegrits committed:
First cut at pivot list plugin back port for 3.0

cheesegrits committed:
Adding safeQuote and safeNameQuote to string helper

cheesegrits committed:
Small change to getContentTemplate() helper, added optional second arg,
$part, allows specifying 'intro', 'full' or 'both'. Default is 'both',
which is the current behavior of concatenating the intro and full
article text into a single response.

cheesegrits committed:
Fixing issue with autocomplete JS, with event on keydown not testing to
see if the autocomplete was the active element, ended up stopping enter
keys in textareas.

cheesegrits committed:
Added additional CSS classes to pivot table plugin.

cheesegrits committed:
Added optional sorting on pivot sum to pivot list plugin.

cheesegrits committed:
Fixing tabs template, was missing default_repeatgroup.php.

trob committed:
Typo in _validationErr

cheesegrits committed:
Fixing 'all' issue with list display.

cheesegrits committed:
Added always safe quoting of values in getFilterVlue() for IN and NOT IN
conditions.

cheesegrits committed:
Merge branch 'master' of ssh://github.com/Fabrik/fabrik

cheesegrits committed:
Adding number formatting for totals in list pivot plugin

cheesegrits committed:
Defensive coding in string element for ltrimword

trob committed:
Fabrik3+ needs renderListData

cheesegrits committed:
Having to commit empty SMS gateway adapter for Global SMS, just to keep
git happy.

cheesegrits committed:
Merge branch 'master' of ssh://github.com/Fabrik/fabrik

cheesegrits committed:
Adding insert ID for PK element in to model's fullFormDataWithTableName
after a new record creation, to keep placeholders happy in email plugin.

pollen8 committed:
fixed: dbjoin as checkbox. Dropdown filter was not respecting the "join where" statement

pollen8 committed:
pivot plugin. remove html from selected Sum column before trying to sum the values. code style

pollen8 committed:
fixed: autocmplete dbjoin using keys to navigate. Pressing enter produced a js error and no selection was made

pollen8 committed:
fixed csv export of list filtered on dates with formatting other than
default db format

pollen8 committed:
fixed: advanced filter then csv export was not working. Swapped from
injecting the filter value to using the filter rawvalue to avoid double
quoting in the main php list query

cheesegrits committed:
Fixing issue with picklist not displaying "to" values on edit / display.

cheesegrits committed:
Merge branch 'master' of ssh://github.com/Fabrik/fabrik

cheesegrits committed:
Fixing snafu with listfilter-min.js.

pollen8 committed:
ensuring db join default eval works

pollen8 committed:
added option to toggle ON/OFF DISTINCT in list select query

pollen8 committed:
fixed pivot viz if sul field had nulber formatting already applied
